src/composite_operators.jl

```diff
@@ -38,9 +38,9 @@ mul!(Y::AbstractVecOrMat, L::DiffEqScaledOperator, B::AbstractVecOrMat) =
 ldiv!(Y::AbstractVecOrMat, L::DiffEqScaledOperator, B::AbstractVecOrMat) =
   lmul!(1/L.coeff, ldiv!(Y, L.op, B))
 factorize(L::DiffEqScaledOperator) = L.coeff * factorize(L.op)
-for fact in (:lu, :lu!, :qr, :qr!, :cholesky, :cholesky!, :ldlt, :ldlt!, 
```

Comment:
please don't mix formatting changes in with other PRs.

Reply:
I'm not sure what you mean, but regardless, I don't recall intending to change any of these lines of code that are highlighted here.